### PR TITLE
Add some missing style and minor bugfix

### PR DESCRIPTION
**Pull request by ASDosjani**
I added the missing styles to the selectable lists, those white boxes in dark mode just hurt my eyes so I made this fix. The `Larson scanner` and the `Blink always` overlays didn't worked with each other properly so they toggle each other, only one of them can be used at a time. I noticed that the channel based color modifier doesn't have default value so I nominated the Throttle for this. Also updated deprecated .change() function.

~~While I was making this fix it stood out that the switch animations doesn't work if I select anyting on the Led table. It is not related to my code, I tried to fix it but I couldn't. If anybody has time I would appreciate a fix for this.~~
Edit: I could fix it.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by KarateBrot** [comment]
You can reopen the PR when you push something again

**Comment by ASDosjani** [comment]
> You can reopen this PR when you push something again

I did, nothing changed.

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:513]:
```suggestion
                if (that.is('.function-o')) {
```

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:516]:
```suggestion
                    if (blink.is(':checked')) {
```

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:523]:
```suggestion
                else if (that.is('.function-b')) {
```

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:526]:
```suggestion
                    if ($('.checkbox .function-o').is(':checked')) {
```

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:1147]: 3 what?

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: src/js/tabs/led_strip.js
Comment (ASDosjani) [reply src/js/tabs/led_strip.js:1147]: 3 is the index of the throttle in the list.

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:520]:
```suggestion
                } else if (that.is('.function-b')) {
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Sorry missed that one in previous review

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [reply src/js/tabs/led_strip.js:1143]: Please be more specific, What list?

Comment (ASDosjani) [reply src/js/tabs/led_strip.js:1143]: image
This one

File: src/js/tabs/led_strip.js
Comment (ASDosjani) [reply src/js/tabs/led_strip.js:520]: I'm sorry, I will pay attention to the brackets.

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:103]: .change is deprecated.
.on('change', function() {}); is preferred.

Comment (ASDosjani) [reply src/js/tabs/led_strip.js]: Done

**Review by haslinghuis — DISMISSED** [review]

**Review by blckmn — APPROVED** [review]

File: src/js/tabs/led_strip.js
Comment (Asizon) [inline src/js/tabs/led_strip.js:506]: Missing parentesis here )

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/js/tabs/led_strip.js
Comment (ASDosjani) [reply src/js/tabs/led_strip.js]: You mean space, right?

**Review by Asizon — DISMISSED** [review]

**Review by haslinghuis — DISMISSED** [review]

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:872]:
```suggestion
        $('.overlays').toggle(!areOverlaysActive(activeFunction);
        $('.modifiers').toggle(!areModifiersActive(activeFunction);
        $('.blinkers').toggle(!areBlinkersActive(activeFunction);
        $('.warningOverlay').toggle(!isWarningActive(activeFunction);
        $('vtxOverlay').toggle(!isVtxActive(activeFunction);
```
Need to check !

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: src/js/tabs/led_strip.js
Comment (ASDosjani) [reply src/js/tabs/led_strip.js:872]: It's not working because this function gets called on every change on switches, I tested.

Comment (haslinghuis) [reply src/js/tabs/led_strip.js:872]: My bad, need to remove the negation ( ! ) and also remove the lines https://github.com/betaflight/betaflight-configurator/pull/2996/files#diff-7fa9d993f9cf0be01c067a90e4edb501336ad7178b3a1a3106db0f6b7eb2e356L857-L871

Comment (haslinghuis) [reply src/js/tabs/led_strip.js:872]: image

Comment (haslinghuis) [reply src/js/tabs/led_strip.js]: Using toggle is a much better and dense solution,

Comment (ASDosjani) [reply src/js/tabs/led_strip.js]: > 

I will check

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Comment by SupaflyFPV** [comment]
Mac works as expected...

**Review by Asizon — APPROVED** [review]

**Pull request merged**
